src/OrbitGl/Button.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "Button.h"
+
+#include "AccessibleCaptureViewElement.h"
+
+namespace orbit_gl {
+
+Button::Button(CaptureViewElement* parent, const Viewport* viewport, const TimeGraphLayout* layout)
+    : CaptureViewElement(parent, viewport, layout) {}
+
+void Button::SetHeight(float height) {
+  if (height == height_) return;
+
+  height_ = height;
+  RequestUpdate();
+}
+
+std::unique_ptr<orbit_accessibility::AccessibleInterface> Button::CreateAccessibleInterface() {
+  // TODO: Should be `AccessibleButton`
```

Comment:
This TODO will be addressed in one of the next PRs, that's why I didn't create a bug for it.